diff -ur faac.orig/Makefile.am faac/Makefile.am
--- faac.orig/Makefile.am	Wed Jul 28 05:18:20 2004
+++ faac/Makefile.am	Fri Nov 28 23:04:33 2008
@@ -1 +1 @@
-SUBDIRS = include libfaac common frontend
+SUBDIRS = include libfaac
diff -ur faac.orig/configure.in faac/configure.in
--- faac.orig/configure.in	Mon Aug 20 15:03:22 2007
+++ faac/configure.in	Fri Nov 28 23:04:33 2008
@@ -76,10 +76,4 @@
 	 #include <netinet/in.h>])
 AC_CHECK_MEMBERS(fpos_t.__pos,,, [#include <stdio.h>])
 
-AC_OUTPUT([
-common/Makefile
-common/mp4v2/Makefile
-libfaac/Makefile
-frontend/Makefile
-include/Makefile
-Makefile])
+AC_OUTPUT(libfaac/Makefile include/Makefile Makefile)
diff -ur faac.orig/include/faac.h faac/include/faac.h
--- faac.orig/include/faac.h	Sun Jul  4 09:12:05 2004
+++ faac/include/faac.h	Fri Nov 28 23:05:25 2008
@@ -26,7 +26,7 @@
 extern "C" {
 #endif /* __cplusplus */
 
-#ifdef WIN32
+#ifdef _WIN32
 # ifndef FAACAPI
 #  define FAACAPI __stdcall
 # endif
